### PR TITLE
fix: Add missing README.md files

### PR DESCRIPTION
**Pull request by kayano**
Adds missing README.md files for charts relayer-exporter and chainpulse.
Fixes typos.

### CODE REVIEW

File: charts/endpoint-controller/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Kubernetes 1.16+
 ## Get Repository Info
 
 ```console
-helm repo add prometheus-community https://archway-network.github.io/helm-charts
```
Comment (shahbazn) [inline charts/endpoint-controller/README.md:14]: nice catch :)

File: charts/hermes-relayer/Chart.yaml [hunk 2]
```diff
@@ -3,5 +3,5 @@ apiVersion: v2
 name: hermes-relayer
 description: A Helm chart for Kubernetes
 type: application
-version: 1.3.1
+version: 1.3.2
 appVersion: 1.5.1
```
Comment (shahbazn) [inline charts/hermes-relayer/Chart.yaml]: Lets please update the default here

Comment (kayano) [reply charts/hermes-relayer/Chart.yaml]: https://github.com/archway-network/helm-charts/pull/11/commits/aa2f2d108289e61192dda5ed5e64bf631e008e74

**Review by shahbazn — APPROVED** [review]
lgtm

**Pull request merged**